docs/index.adoc

```diff
@@ -5284,6 +5392,12 @@ By default, the usage help message only shows the subcommands of the specified c
 and not the nested sub-subcommands. This can be customized by specifying your own <<Reordering Sections,`IHelpSectionRenderer`>> for the command list section.
 The `picocli-examples` module has an https://github.com/remkop/picocli/blob/master/picocli-examples/src/main/java/picocli/examples/customhelp/ShowCommandHierarchy.java[example] that shows how to accomplish this.
 
+If you want to get usage help for nested subcommands, make sure that all commands in the hierarchy have `picocli.CommandLine.HelpCommand` as (one or their) subcommands. Then you can get help message for a nested subcommand like this:
```

Comment:
That is not the only way though. How about this:

> To get usage help for nested subcommands, either set `mixinStandardHelpOptions = true` in all commands in the hierarchy, or add `picocli.CommandLine.HelpCommand` to the subcommands of all commands.

Reply:
You are right. There are two ways to get use help, so we should document both of them. Your proposal is perfect, I changed the PR accordingly.

Concerning the `bash`-block showing how to invoke usage help:
There was a previous [discussion](https://github.com/remkop/picocli/issues/566#issuecomment-453351720) that teached me that the way kow to invoke usage help is not necessarily self-explaining. That's why added the `bash`-block and IMHO, we should keep it. I modified the code block to take both ways of getting usage help into account. I agree it's a bit lengthy now, but I would like to keep that block, nevertheless.

Thanks for reviewing my modified PR!